common/src/main/java/org/apache/atlas/service/redis/RedisServiceLocalImpl.java

```diff
@@ -1,44 +0,0 @@
-package org.apache.atlas.service.redis;
```

Comment:
local server startup will fail in case we remove this class, I think you should update everyone on channel the workaround to run locally to change getProdConfig() & getCacheImplConfig() -> getLocalConfig()